tests/convergence/BAE.cpp

```diff
@@ -0,0 +1,393 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+// Compile with: make BAE
```

Comment:
I'd prefer we stick with lowercase-only filenames: `BAE.cpp` --> `bae.cpp`. For example, `fe.cpp` is lowercase. 😄

Reply:
In light of https://github.com/mfem/mfem/issues/1254: do we want to commit this?

Reply:
> I'd prefer we stick with lowercase-only filenames: `BAE.cpp` --> `bae.cpp`. For example, `fe.cpp` is lowercase. 😄

Sure...I renamed it

Reply:
> In light of #1254: do we want to commit this?

I don't know...we kind of forgot about this for a while. Whatever you guys prefer, I am fine with it